examples/httpPlusdb/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM golang:1.21.4
 WORKDIR /app
-COPY . .
+COPY ./main.go ./main.go
```

Comment:
Currently examples are only one file apps, however that might change, should not we copy all go files
```
COPY ./*.go *.go
```